packages/esm-service-queues-app/src/queue-table/queue-table-metrics.component.tsx

```diff
@@ -1,29 +1,20 @@
-import React, { useCallback } from 'react';
-import { useTranslation } from 'react-i18next';
-import { useQueueEntries, useQueueEntriesMetrics } from '../hooks/useQueueEntries';
-import QueueTableMetricsCard from './queue-table-metrics-card.component';
+import React from 'react';
 import styles from './queue-table-metrics.scss';
-import { type Queue } from '../types';
+import type { Concept, Queue } from '../types';
+import { ExtensionSlot } from '@openmrs/esm-framework';
 
 interface QueueTableMetricsProps {
   selectedQueue: Queue;
+  selectedStatus: Concept;
 }
 
```

Comment:
This is where we need to figure out a way to render extensions based on what extensions are configured on the slot, and what statuses are available on the current queue.  We _may_ be able to use the new `useRenderableExtensions` function from esm-framework (esm-react-utils) to achieve this.  I'd be interested in @chibongho  and @ibacher given the similar work they recently tried to achieve with the queue table.